cinema/views.py

```diff
@@ -0,0 +1,37 @@
+from django.shortcuts import render
+from rest_framework.response import Response
+from rest_framework import status
+from cinema.models import Movie
+from cinema.serializer import MovieSerializer
+from rest_framework.decorators import api_view
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    elif request.method == "POST":
+        serializer = MovieSerializer(data=request.data)
+        if serializer.is_valid():
+            serializer.save()
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+@api_view(["GET", "PUT", "DELETE"])
+def movie_detail(request, pk):
+    movie = Movie.objects.get(pk=pk)
```

Comment:
Consider adding error handling for the case where a `Movie` with the given primary key does not exist. You can use `get_object_or_404` from Django shortcuts to handle this gracefully and return a 404 response if the movie is not found.